contracts/interfaces/IAelinPool.sol

```diff
@@ -12,5 +12,25 @@ interface IAelinPool {
         uint256 purchaseDuration;
         address[] allowListAddresses;
         uint256[] allowListAmounts;
+        NftData[] nftData;
+    }
+
+    // collectionAddress should be unique, otherwise will override
+    struct NftData {
```

Comment:
something more explicit and readable than NftData, like `NftCollectionRules` or `NFTCollectionRules`